src/ElmFive.elm

```diff
@@ -181,27 +181,23 @@ translations =
 
 translate : String -> String
 translate language =
-    case Dict.get language translations of
-        Nothing ->
-            "five"
-
-        Just translation ->
-            translation
+    Dict.get language translations
+        |> Maybe.withDefault "five"
 
 
 isFive : Int -> Bool
-isFive value =
-    value == five
+isFive =
+    (==) five
```

Comment:
We have a general rule that, if the parameter is clear as to what it is, then we go point-free. It initially feels clever, until you realize that `==` is actually a function, set as infix. So, `(==)` is actually a function that takes two parameters.